src/SonarScanner.MSBuild.Common/Resources.Designer.cs

```diff
@@ -8,492 +8,502 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace SonarScanner.MSBuild.Common; 
```

Comment:
When you edit the Designer from VS, it converts this to a normal namespace, not a file-scoped one.

I suggest we leave it as-is, as you rarely (or never) touch this manually.